app/scss/components/_grid.scss

```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```

Comment:
You don't need them at all. I remember telling you our grid classes are just like that of ionic at http://ionicframework.com so you could have `.col-40` and `.col-60` or `.col-40` and `.col`. With the benefit of our own grids automatically adjusting to the space left and given `.col` a 60% width. I explained all this to you. This is what makes our grid way better than bootstrap grid. Don't edit this grid partial at all